sealtk/core/test/TestTracks.cpp

```diff
@@ -37,9 +37,12 @@ void testTrackData(QAbstractItemModel const& model,
 
   auto const modelHasClassification =
     !model.data(index, ClassificationRole).isValid();
-  if (state.classification.isEmpty() == !modelHasClassification)
+  if (!modelHasClassification)
+  {
+    QVERIFY(state.classification.isEmpty());
+  }
+  else
   {
-    qDebug() << state.classification << model.data(index, ClassificationRole);
```

Comment:
This removal was not in the same commit that introduced it in the first place. I'm OK with not having a "pure" history, so it's up to you if you want to fix it.

Reply:
Ack, no, I'll fix it... possibly I squashed the fixup commit into the wrong spot :cry:.

Reply:
Fixed; thank you for noticing!!!